user_interaction/themes.py

```diff
@@ -75,9 +75,9 @@ class ScalaUserTheme(SquireUserTheme):
     css = ('themes/scala-theme.css',)
 
 
-class 日本スキン(SquireUserTheme):
+class KinjinUserTheme(SquireUserTheme):
```

Comment:
But I wanted to troll the other programmers too. 😭

Reply:
Sorry, Eric made me change this